client/src/test/java/io/pravega/keycloak/client/KeycloakAuthzClientTest.java

```diff
@@ -195,6 +229,15 @@ private static String getResourceFile(String resourceName) {
         return new File(KeycloakAuthzClientTest.class.getClassLoader().getResource(resourceName).getFile()).getAbsolutePath();
     }
 
+    private static String getResourceString(String resourceFilePath) {
+        try {
+            return new String(Files.readAllBytes(Paths.get(resourceFilePath)));
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Minor but you usually don't want to swallow errors. Replace with `throw new RuntimeException(e);`.